### PR TITLE
afaryy/c02-network01

### PR DESCRIPTION
**Pull request by afaryy**
# Description

Exercise (e.g. C01-E01): c02-network01

Student Name: Yvonne Yao(afaryy)

> A brief explanation of what has been done.

## Type of change

- [X] Exercise Submission (one exercise per PR)
  - [X] This PR is just for **one** exercise of a class
  - [X] PR name follows the pattern `<github-username>/<exercise-number>`
  - [X] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [X] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/02class/exercises/c02-network01/afaryy/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  - IP in binary: `11111001.10100101.10100110.100001 11`
+  - Netmask in binary: `11111111.11111111.11111111.111111 00` 
+  - Netmask in decimals: `255.255.255.252`
+  - private/public: `public`
+  - hosts: `2` (249.165.166.133-249.165.166.134)
+
+- Answers for `236.68.223.18/32`:
+
+  - IP in binary: `11111001.10100101.10100110.100001 11`
```
Comment (roperto) [inline classes/02class/exercises/c02-network01/afaryy/README.md:31]: This one is incorrect (that was from the previous item).

File: classes/02class/exercises/c02-network01/afaryy/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  - Netmask in binary: `11111111.11111111.11111111.11111111` 
+  - Netmask in decimals: `255.255.255.255`
+  - private/public: `public`
+  - hosts: `1` (236.68.223.19)
```
Comment (roperto) [inline classes/02class/exercises/c02-network01/afaryy/README.md:35]: `/32` is kind-of a special case. It does not declare a subnet but a single-host instead.

This is not a network and the only host would be `236.68.223.18` itself.

When we use something different than `/32` we mean "all IPs in that network (or subnet for short)".

When we use `/32` we mean "only that specific IP address"

Comment (afaryy) [reply classes/02class/exercises/c02-network01/afaryy/README.md:35]: Thank you for your explaination.

File: classes/02class/exercises/c02-network01/afaryy/README.md (hunk 1 above)
Comment (afaryy) [reply classes/02class/exercises/c02-network01/afaryy/README.md:31]: I am a little careless.I will correct it. Thanks.

**Pull request merged**